### DISCUSSION TITLE
Install Alpine

### DISCUSSION THREAD

**Discussion by 3ncelado** [alpinejs/alpine#3551] [1. Help]
Hi, I hope you can help me and I would appreciate it. I would like to delve deeper into Alpine.js, but I don't use Node and I'm not that fond of CDNs. Is it possible to copy the CDN code into a local JS file and call it locally?

**Comment by SimoTod** [comment]
Yep, go for it

**Reply by 3ncelado** [reply]
ty bro :3

**Comment by ekwoka** [comment]
Yes, just download it.

**Reply by 3ncelado** [reply]
ty, bro
